commands/fcp/addlog.js

```diff
@@ -0,0 +1,71 @@
+import { SlashCommandBuilder, EmbedBuilder } from 'discord.js';
+import fetch from 'node-fetch';
+import { Config } from '../../config/config.js';
+import { sendToSentry } from '../../utils.js';
+
+const config = new Config();
+
+export default {
+  data: new SlashCommandBuilder()
+    .setName('add-log')
+    .setDescription('Add a FCP audit log to a member')
+    .addUserOption(option =>
+      option
+        .setName('user')
+        .setDescription('The user to add the log to')
+        .setRequired(true)
+    )
+    .addStringOption(option =>
+      option
+        .setName('log-reason')
+        .setDescription('Log content')
+        .setRequired(true)
+    ),
+  async execute(interaction) {
+    try {
+      const user = interaction.options.getUser("user");
+      const logReason = interaction.options.getString("log-reason");
+
+      const response = await fetch(`https://flightcrew-beta.thepilotcub.org/api/users/find/${user.id}/audit-logs/new`, {
```

Comment:
Please add the fcp base url as the first part of your link. If you set it within your .env file, you will be able to define the beta. This will help on the back end when I go to use some k8s magic.